src/modules/main/sidebar/sidebar-links/SidebarLinks.scss

```diff
@@ -0,0 +1,40 @@
+@import 'styles/mixins';
+
+.links {
+    position: relative;
+
+    .link {
+      @include rowCenter;
+
+      button {
+        width: 100%;
+        padding: 0;
+        border-radius: 0;
+        color: $primary;
+
+        & > span {
+          @include col;
+
+          & > span {
+            margin-top: 6px;
+            @include linkSmall;
```

Comment:
@include always should be first

Reply:
This code is not my, I transferred it from another file, but I fixed it